CONTRIBUTING.md

```diff
@@ -74,19 +74,12 @@ We want your work to be readable by others; therefore, we encourage you to note
 
 - We encourage the use of Python [f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) where they make the code easier to read.
 
-- Please consider running [__psf/black__](https://github.com/python/black) on your Python file(s) before submitting your pull request.  This is not yet a requirement but it does make your code more readable and automatically aligns it with much of [PEP 8](https://www.python.org/dev/peps/pep-0008/). There are other code formatters (autopep8, yapf) but the __black__ formatter is now hosted by the Python Software Foundation. To use it,
-
-  ```bash
-  python3 -m pip install black  # only required the first time
-  black .
-  ```
-
-- All submissions will need to pass the test `flake8 . --ignore=E203,W503 --max-line-length=88` before they will be accepted so if possible, try this test locally on your Python file(s) before submitting your pull request.
-
-  ```bash
-  python3 -m pip install flake8  # only required the first time
-  flake8 . --ignore=E203,W503  --max-line-length=88 --show-source
-  ```
+- Do not forget that running `pre-commit` checks locally will help you to follow code quality standards.
```

Comment:
This is indecipherable for first-time contributors.  We would need a link to pre-commit instructions on how to install and initialized that tool.  I think that will be beyond the reach of many newbies.

Reply:
Not sure what to do?  Just remove this line? Should I revert deleting mentions about other tools?